blocks/README.md

```diff
@@ -2,15 +2,59 @@
 [github_star]: https://github.com/hashintel/hash/tree/main/blocks#
 [discord]: https://hash.ai/discord?utm_medium=organic&utm_source=github_readme_hash-repo_blocks
 
+[`calculation`]: calculation
+[`callout`]: callout
+[`chart`]: chart
+[`code`]: code
+[`countdown`]: countdown
+[`divider`]: divider
+[`drawing`]: drawing
+[`embed`]: embed
+[`github-pr-overview`]: github-pr-overview
+[`header`]: header
+[`html-para`]: html-para
+[`image`]: image
+[`paragraph`]: paragraph
+[`person`]: person
+[`shuffle`]: shuffle
+[`stopwatch`]: stopwatch
+[`table`]: table 
+[`timer`]: timer
+[`toggle-item`]: toggle-item
+[`video`]: video
+
 [![github_banner](https://hash.ai/cdn-cgi/imagedelivery/EipKtqu98OotgfhvKf6Eew/5a38c5f3-6474-4b6c-71e6-ecf01914f000/github)][github_banner]
 
 [![discord](https://img.shields.io/discord/840573247803097118)][discord] [![github_star](https://img.shields.io/github/stars/hashintel/hash?label=Star%20on%20GitHub&style=social)][github_star]
 
-# HASH Blocks
+# Blocks
```

Comment:
@CiaranMn Would welcome your help fleshing out the spec target/status of these blocks. I am unclear re: `html-para` and _believe_ we intend to replace `table` with Yusuf's in-flight one (although that could be considered a rather dramatic update to this existing one, rather than the existing one being unmaintained as implied below).

Reply:
I would:
- mark `html-para` as unmaintained, we can remove it in a bit
- I'd consider the same for `toggle-item`

Reply:
Actioned in https://github.com/hashintel/hash/commit/9bce6f2774f24b5fc65d3fc88638a8f9fe3cd353 - thank you.